src/model.rs

```diff
@@ -105,16 +105,16 @@ impl GuestLanguage {
 
     pub fn tier(&self) -> GuestLanguageTier {
         match self {
-            GuestLanguage::Rust => GuestLanguageTier::Tier2,
-            GuestLanguage::Go => GuestLanguageTier::Tier2,
-            GuestLanguage::C => GuestLanguageTier::Tier2,
-            GuestLanguage::Zig => GuestLanguageTier::Tier3,
-            GuestLanguage::JavaScript => GuestLanguageTier::Tier2,
+            GuestLanguage::Rust => GuestLanguageTier::Tier1,
+            GuestLanguage::Go => GuestLanguageTier::Tier1,
+            GuestLanguage::C => GuestLanguageTier::Tier1,
+            GuestLanguage::Zig => GuestLanguageTier::Tier2,
+            GuestLanguage::JavaScript => GuestLanguageTier::Tier1,
             GuestLanguage::CSharp => GuestLanguageTier::Tier3,
```

Comment:
in instruction C# was changed to tier2, is this correct?

Reply:
Yes that's correct. There's only two kind of adapters. C# is supposed to be tier 2 but something is broken or unimplemented in it so it can't even read stdin that's why it got "downgraded" to tier 3